### PR TITLE
add link to eslint rule in message

### PR DESCRIPTION
**Pull request by granteagon**
User can now click the link in the linter message to see the rule
definition on the eslint site.

Also added a missing space between the rule span and the rule message.

The only side-effect I'm aware of is that css for my theme styles links
close in color to the badge, as seen in this screenshot:
http://cl.ly/2O2o0Y2o0O3L. However, the rule at the bottom looks good,
again in my theme (One Dark).

### CODE REVIEW

File: src/main.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 'use babel'
 
 import { CompositeDisposable, Range } from 'atom'
-import { spawnWorker, showError } from './helpers'
+import { spawnWorker, showError, ruleURI } from './helpers'
 import escapeHTML from 'escape-html'
 
+
+
```
Comment (Arcanemagus) [inline src/main.js]: Extra blank line, causing a lint issue.

**Comment by Arcanemagus** [comment]
Other than the lint issue it looks good from a glance at the code, checking it out to test locally.

**Comment by granteagon** [comment]
Yeah, I just noticed that.  Will fix and commit in just a sec.  The color issue I mentioned seems to be down to the theme.  So I will try and fix over on the One Dark theme repo.

**Comment by Arcanemagus** [comment]
Hmmm, it looks off on the Seti theme as well, but not as bad:
image
(Btw, you can paste images directly here)

@steelbrain is this something that can be fixed in linter-ui-default, or should we not care about that since it's a theme issue?

**Comment by Arcanemagus** [comment]
Huh, was wondering why the specs _weren't_ broken by this, turns out we don't have specs testing when "Show Rule ID in Messages" is enabled :confused:.

**Comment by Arcanemagus** [comment]
Btw, this needs to be rebased on master (which will update the dist files so `worker-helpers.js` is no longer in this PR).

**Comment by granteagon** [comment]
@Arcanemagus Happy to create a spec for that if need be.  Not sure if you're saying I should, or if there should already have been one.  The build had already failed when I forked the repo.

Also, can you give me the command to rebase?  I don't do that very often.

**Comment by Arcanemagus** [comment]
I'm saying there should have already been one :wink:.

It should be as simple as `git rebase master`, but you have a conflicting commit in there so you will need to resolve that manually. ([merge-conflicts](https://atom.io/packages/merge-conflicts) is _very_ useful for that.)

**Comment by Arcanemagus** [comment]
Travis failing is due to the huge `babel-eslint` + `eslint@2.3.0` mess, AppVeyor looks to be a bizarre cache issue, trying to restart that build.

**Comment by granteagon** [comment]
@Arcanemagus we should probably create a different pull request for adding the missing spec–just for cleanliness–yes? 

Rebased, btw.

**Comment by Arcanemagus** [comment]
Looks like you did a `git merge master`, the difference between what that did and what `git rebase master` would have done is probably best shown graphically:
image

`git rebase master` changes where your "base" of the branch is (the start of your line there), `git merge master` just makes your branch be in the same state as master, but the tree isn't the same.

You need to:
1. Remove that latest commit (pretty simple in GitHub Desktop, or you can do it with `git rebase -i HEAD~2` and then commenting out the latest commit).
2. `git rebase master`
3. Fix merge conflicts (the Atom package above makes that pretty easy)
4. `git rebase --continue`

You should end up with no merge conflicts, and no "merge commit", you'll need to do a `git push -f` to push it up to the remote.

**Comment by Arcanemagus** [comment]
And yes, that should be a different PR.

**Comment by granteagon** [comment]
I screwed something up.  I'll just close this PR and do a new one from a fresh branch.

**Comment by Arcanemagus** [comment]
If you ever need help we're on Atom's Slack instance btw :wink:

**Pull request closed without merging**